node/silkworm/common/settings.hpp

```diff
@@ -29,14 +31,19 @@
 namespace silkworm {
 
 struct NodeSettings {
+    boost::asio::io_context asio_context;            // Async context (e.g. for timers)
     std::unique_ptr<DataDirectory> data_directory;   // Pointer to data folder
     db::EnvConfig chaindata_env_config{};            // Chaindata db config
     uint64_t network_id{kMainnetConfig.chain_id};    // Network/Chain id
+    std::optional<ChainConfig> chain_config;         // Chain config
     size_t batch_size{512_Mebi};                     // Batch size to use in stages
     size_t etl_buffer_size{256_Mebi};                // Buffer size for ETL operations
     std::string private_api_addr{"127.0.0.1:9090"};  // Default API listener
-    bool fake_pow{false};                            // Whether to check for PoW or not
+    std::string sentry_api_addr{};                   // Default address(es) of sentry
+    bool fake_pow{false};                            // Whether to verify Proof-of-Work
```

Comment:
Has no effect at the moment. Perhaps we can remove `fake_pow` for now and add it later need be.